src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/IdentifiersShouldHaveCorrectSuffixTests.cs

```diff
@@ -1756,15 +1757,19 @@ public class SomeSubSubClass : SomeSubClass {}"
                 }
             };
 
+#if NETCOREAPP3_1
+            if (editorConfigText.Contains("exclude_indirect_base_types = false", StringComparison.Ordinal))
```

Comment:
❔ Why not do this for both cases? Same applies to the case below.

Reply:
Because the overload with the `StringComparison` does not exist for non-netcore platforms.

Reply:
Perhaps an extension method?